### PR TITLE
Linter GitHub workflow action

### PR DESCRIPTION
**Pull request by waterseeker**
I ran npm install for prettier and eslint, with eslint I chose the airbnb style but I can change that if needed. I also created a new linging.yml file that should run as an action on push to the github repo.

Fixed linting issues:
replaced underscore naming with camel case
changed double quotes to single

### CODE REVIEW

**Comment by waterseeker** [comment]
Running into issues getting prettier and eslint to cooperate. I'll submit the request again once I get it straightened out.

**Comment by r33beers** [comment]
Maybe ignore prettier for now? eslint can be good enough.

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  env: {
+    browser: true,
```
Comment (r33beers) [inline .eslintrc.js]: None of the code you have here runs in the browser. this shouldn't be set to true.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    es6: true
+  },
+  extends: ["airbnb-base"],
+  globals: {
+    Atomics: "readonly",
+    SharedArrayBuffer: "readonly"
```
Comment (r33beers) [inline .eslintrc.js]: What do these do?

File: .eslintrc.js (hunk 1 above)
Comment (waterseeker) [reply .eslintrc.js]: I think this is just from the CLI set-up config. I think all/most of these changes are going to be trashed when I get back to trying to get this to work.

File: .eslintrc.js (hunk 2 above)
Comment (waterseeker) [reply .eslintrc.js]: airbnb is a styleguide picked from the CLI setup, I don't know what the globals do. They're probably just from the setup.

**Pull request closed without merging**